### ISSUE TITLE
explanation of Default Role None (deny access)

### ISSUE THREAD

**Issue by chaplina** [heiglandreas/authLdap#50]
Assumes authLDAPGroupEnable is false (not checked) and Default Role is set to None (deny access).

If a WP user exists with the username provided, and that user is in LDAP, authLDAP will succeed with the correct password.

If a WP user does not exist with the username provided, and that user is in LDAP, authLDAP will not succeed.

We are using this in our multisite successfully to allow LDAP authentication for Editors for publicly accessible sites.

**Comment by heiglandreas** [comment]
Is this behaviour a problem for you? I can't really tell from the description :cry:

**Comment by chaplina** [comment]
It works but the docs should be updated to better explain the workflow.

On Apr 1, 2015 12:02 PM, Andreas Heigl <notifications@github.com> wrote:

Is this behaviour a problem for you? I can't really tell from the description [:cry:]

**Comment by heiglandreas** [comment]
You are absolutely right on that one. There are a lot of things in the docs that need updates after those last few weeks.  I think I'll use the easter-time to do some work on them.

 Thanks for reminding me though. :wink:
